python_modules/libraries/dagster-dg/dagster_dg/utils/telemetry.py

```diff
@@ -0,0 +1,110 @@
+import datetime
+import sys
+from collections.abc import Mapping
+from functools import wraps
+from typing import Any, Callable, Optional, TypeVar, Union, overload
+
+import click
+from dagster_shared.telemetry import (
+    TelemetrySettings,
+    get_or_set_instance_id,
+    get_telemetry_enabled_from_dagster_yaml,
```

Comment:
it seems weird that the only way to turn off telemetry in dg is from the dagster.yaml? Should we have a dg-specific way to opt out as well? kind of like how you had a dg place to stick cloud auth tokens and a dagster-cloud-cli place to stick auth tokens

Reply:
Added a setting to `~/.dg/config.yaml` - the CLI checks that it is not disabled there or in the dagster.yaml